src/main/java/io/supertokens/storage/postgresql/FlywayMigration.java

```
@@ -35,25 +33,79 @@ public static void startMigration(Start start) {
-               the migration. After that, check how to store the baselineversion in database. Flyway might have
-               something.
-            */
             String baselineVersion = BaselineMigrationQueries.getBaselineMigrationVersion(start);
-            boolean access_token_signing_key_dynamic = true; //Add from Config.getBaseConfig(this);
-            String location = "classpath:/db/migration/access_token_signing_key_dynamic_" + access_token_signing_key_dynamic;
 
             Flyway flyway = Flyway.configure()
                     .dataSource(ConnectionPool.getHikariDataSource(start))
                     .baselineOnMigrate(true)
                     .baselineVersion(baselineVersion)
-                    .locations("classpath:/db/migration/common", location)
+                    .locations("classpath:/io/supertokens/storage/postgresql/migrations")
+                    .placeholders(getPlaceholders(start))
                     .load();
             flyway.migrate();
         } catch (Exception e) {
             Logging.error(start, "Error Setting up Flyway.", true);
+            Logging.error(start, e.toString(), true);
            // TODO: Find all possible exception
         }
     }
+
+    @TestOnly
+    public static void executeTargetedMigration(Start start, String migrationTarget) {
+        try {
+        Flyway flyway = Flyway.configure()
+                .dataSource(ConnectionPool.getHikariDataSource(start))
+                .target(migrationTarget)
+                .locations("classpath:/io/supertokens/storage/postgresql/migrations")
+                .baselineOnMigrate(true)
+                .placeholders(getPlaceholders(start))
+                .load();
+        flyway.migrate();
+        } catch (Exception e) {
+            Logging.error(start, "Error Setting up Flyway.", true);
+            Logging.error(start, e.toString(), true);
+            // TODO: Find all possible exception
```

For tests, we do not have Logging. Just let the exception be thrown so the test would just fail?